commit 120990a0a1ca5ceeb311fa7ce511ce0002f022ed (refs/changes/04/2504/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-09-06T14:19:09-07:00 (2 years, 1 month ago)
    
    DOCKER-1097 docker pull fails for registry server that has no auth setup
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 49d1fd5..ec9f7fb 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # IMGAPI changelog
 
+## 4.0.5
+
+- DOCKER-1097 docker pull fails for registry server that has no auth setup
+
 ## 4.0.4
 
 - joyent/sdc-imgapi#13 imgapi crash when pulling from docker.io with invalid credentials
diff --git a/package.json b/package.json
index 24062c7..bfdd7f8 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "4.0.4",
+  "version": "4.0.5",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -11,7 +11,7 @@
     "bunyan": "1.8.8",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.8",
+    "docker-registry-client": "3.2.9",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
